### PR TITLE
Merge/release 7.62.2 into release 7.63

### PR DESCRIPTION
**Pull request by mebarbosa**

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR has the <code>releases</code> label: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,9 @@
         ([#2037](https://github.com/Automattic/pocket-casts-android/pull/2037))
     *   Sleep Timer restarts automatically if you play again within 5 minutes
         ([#2048](https://github.com/Automattic/pocket-casts-android/pull/2048))
+*   Updates:
+    *   Updated widget and added two new variants - small and large.
+        ([#2043](https://github.com/Automattic/pocket-casts-android/pull/2043))
```
Comment (MiSikora) [inline CHANGELOG.md:40]: This isn't correct.

Comment (MiSikora) [reply CHANGELOG.md:40]: We still need to align `version.properties` with the hotifx. Unless you plan to manage version code manually.

**Comment by mebarbosa** [comment]
Closing this PR since there is no difference between these two branches. This changelog one is incorrect so I should remove it

**Pull request closed without merging**